src/common/blockchain.sh

```diff
@@ -168,13 +167,14 @@ function install_fabric_chaincode {
 
     echo "Installing chaincode '$CC_PATH' with id '$CC_ID' and version '$CC_VERSION'..."
     
-    CHAINCODE_FILES=$(find ${CC_PATH} -type f ! -name "*test*")
+    CHAINCODE_FILES=$(find "$CC_PATH" -type f ! -name "*test*")
     CHAINCODE_FILE_OPTS=""
     for CHAINCODE_FILE in ${CHAINCODE_FILES}
     do
         CHAINCODE_FILE_OPTS="${CHAINCODE_FILE_OPTS} -F files[]=@${CHAINCODE_FILE}"
     done
 
+    # shellcheck disable=2086
```

Comment:
Why is this required?

Reply:
The way the CHAINCODE_FILE_OPTS variable is built, in order for it to work within the curl call, it cannot be wrapped in quotes (which shellcheck requires around variables). If you can find a way around this, I will gladly accept it. This was the lazy fix to get the rest of the file compliant at least.

Reply:
Ok, I think I see what the problem is now. I think there may be a better option than multiple `-F` arguments- I'll open an issue to have a look at some point.